modules/xmpp/JingleSessionPC.js

```diff
@@ -1412,16 +1412,15 @@ export default class JingleSessionPC extends JingleSession {
                     ssrcObj.mtype + "\"]>description");
                 if (!desc || !desc.length)
                     return;
-                ssrcObj.ssrc.ssrcs.forEach(function (ssrc) {
+                ssrcObj.ssrcs.forEach(function (ssrc) {
```

Comment:
It doesn't make sense to fix it only in one place of JingleSessinPC and fixing them in whole file is too big effort at this point. I've spent a lot of time on old code fixes anyway.